cardano-cli/test/Test/CLI/Shelley/Golden/Genesis/InitialTxin.hs

```diff
@@ -0,0 +1,34 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Test.CLI.Shelley.Golden.Genesis.InitialTxin
+  ( golden_shelleyGenesisInitialTxin
+  ) where
+
+import Cardano.Prelude hiding (to)
```

Comment:
I think we can remove this `hiding` clause here and also in the other test modules introduced in this PR.

Reply:
Fixed!